pandas/tests/generic/test_finalize.py

```diff
@@ -259,10 +258,9 @@
     pytest.param(
         (
             pd.DataFrame,
-            frame_data,
-            operator.methodcaller("quantile", numeric_only=True),
+            ({"A": [pd.Timedelta(days=1), pd.Timedelta(days=2)]},),
```

Comment:
why did this last change?

Reply:
It was an exact duplicate of the test two tests up, and therefore redundant. I wanted to add this new test, so I just change this one.